src/StyleSheet.js

```diff
@@ -128,40 +126,40 @@ export default class StyleSheet {
   }
 
   /**
-   * Create rules, will render also after stylesheet was rendered the first time.
+   * Create styles, will render also after stylesheet was rendered the first time.
    *
-   * @param {Object} rules name:style hash.
+   * @param {Object} styles name:style hash.
    * @param {Object} [options]
    * @return {Array} array of added rules
    * @api public
    */
-  addRules(rules, options) {
+  addRules(styles, options) {
```

Comment:
same here, addRules accepts styles object and creates rules out of it, they are not rules at the moment user passes them, seemed better to me as we always communicate it as styles to the user land.